CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 - Stop sharing the same attributes hash between the fetched record and the memoized cache, which could interfere with dirty tracking
 
+#### 0.3.2
+
+- Fetch association returns relation or array depending on the configuration
```

Comment:
This isn't clear about how it is configured by default.  It should behave the same way by default until 0.4 is released.